src/MavenUtils.ts

```diff
@@ -46,21 +50,57 @@ export default class MavenUtils {
       let authBlock = ''
       // Check if mavenUser or mavenPassword is to be appended as variable in the authentication bean
       if (isMavenUserVar || isMavenPwdVar) {
-        authBlock = `{ authentication(userName: ${mavenUser!.slice(
-          1,
-          -1
-        )}, password: ${mavenPassword!.slice(1, -1)}) }`
+        authBlock = `
+          credentials {
+            username = ${mavenUser!.slice(1,-1)}
+            password = ${mavenPassword!.slice(1, -1)}
+          }`
       } // Check if mavenUser or mavenPassword is to be appended as value in the authentication bean
       else if (mavenUser || mavenPassword) {
-        authBlock = `{ authentication(userName: "${mavenUser}", password: "${mavenPassword}") }`
+        authBlock = `
+          credentials {
+            username = ${mavenUser}
+            password = ${mavenPassword}
+          }`
       }
       // --config '{"mavenUser": "myUser","mavenPassword": "myPassword"}'
-      // Result : "repository(url: "http://domain.name:8081/repositories") { authentication(userName: "myUser", password: "myPassword") }”
+      // Result :
+      // repositories {
+      //   maven {
+      //     url = "http://domain.name:8081/repositories"
+      //     credentials {
+      //       username = "myUser"
+      //       password = "myPassword"
+      //     }
+      //   }
+      // }
       // --config '{"mavenUser": "[myUserVar]","mavenPassword": "[myPasswordVar]”}'
-      // Result : "repository(url: "http://domain.name:8081/repositories") { authentication(userName: myUserVar, password: myPasswordVar) }”
+      // Result :
+      // repositories {
+      //   maven {
+      //     url = "http://domain.name:8081/repositories"
+      //     credentials {
+      //       username = myUserVar
+      //       password = myPasswordVar
+      //     }
+      //   }
+      // }
       // no config
-      // Result : "repository(url: "http://domain.name:8081/repositories")
-      return `repository(url: "${mavenRepositoryUrl}") ${authBlock}`
+      // Result :
+      // repositories {
+      //   maven {
+      //     url = "http://domain.name:8081/repositories"
+      //   }
+      // }
+
+      // trim() to remove AuthBlock empty line when not needed.
+      var mavenRepository = `url = "${mavenRepositoryUrl}"${authBlock}`.trim()
+
+      return `repositories {
+        maven {
+          ${mavenRepository}
```

Comment:
Please make sure to use standard 4-space indentation for all Groovy code (things that end up in a .gradle file).